src/theme/TOC/index.tsx

```
@@ -0,0 +1,64 @@
+
+export default function TOC({className, metadata, ...props}: Props & {metadata: any }): ReactNode {
+  const url = `https://prisma.io/docs${metadata.slug}`;
+  const externalProps = {
+     target:'__blank',
+     rel:'openeer noreferrer'
+  }
+  const markdown = metadata.editUrl.replace("github", "raw.githubusercontent").replace("/blob", "").replace("/tree", "");
+  return (
+    <div className={styles.tableOfContentsWrapper}>
+      <Dropdown anchorText="Open in" items={[
+        <div onClick={() => getMarkdown(markdown)}><Icon btn="left" icon="fa-brands fa-markdown" size='16px'/>Copy as Markdown</div>,
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)